hcn/hcnloadbalancer.go

```diff
@@ -36,6 +36,7 @@ var (
 	LoadBalancerFlagsNone LoadBalancerFlags = 0
 	// LoadBalancerFlagsDSR enables Direct Server Return (DSR)
 	LoadBalancerFlagsDSR LoadBalancerFlags = 1
+	LoadBalancerFlagsIPv6 LoadBalancerFlags = 2
 )
```

Comment:
Ideally these would be constants also, is there any reason they aren't?

Reply:
Then you can do the same as https://github.com/microsoft/hcsshim/pull/1013/files/d3f1ab7aff4da7a2e367a9a05ab5b2c6c919fcca..f568d433f83efc97f2eaecb882ef8e1e79357f23#r623456888 also

Reply:
I have no idea why they were not made constants in the first place over here :). I would prefer we not change this as these are being used from kubeproxy.

Reply:
Just checking in case anyone knew 😆. Them being constants wouldn't affect anything on them pulling in a new release though, so it should be harmless. They'll be the same type (and value), just not reassignable now.

Reply:
Just code hygiene